File: src/main/java/um/haberes/report/model/haberes/core/CursoCargoNovedadDto.java
package um.haberes.report.model.haberes.core;

import java.math.BigDecimal;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CursoCargoNovedadDto {

    private Long cursoCargoNovedadId;

    private Long cursoId;

    private int anho = 0;

    private int mes = 0;

    private Integer cargoTipoId;

    private Long legajoId;

    private BigDecimal horasSemanales = BigDecimal.ZERO;

    private BigDecimal horasTotales = BigDecimal.ZERO;

    private byte desarraigo = 0;

    private byte alta = 0;

    private byte baja = 0;

    private byte cambio = 0;

    private String solicitud;

    private byte autorizado = 0;

    private byte rechazado = 0;

    private String respuesta;

    private byte transferido = 0;

    private CursoDto curso;

    private CargoTipoDto cargoTipo;

    private PersonaDto persona;

}

File: src/main/java/um/haberes/report/kotlin/dto/haberes/core/CursoCargoNovedadDto.kt
package um.haberes.report.kotlin.dto.haberes.core

import java.math.BigDecimal

data class CursoCargoNovedadDto(

    var cursoCargoNovedadId: Long? = null,
    var cursoId: Long? = null,
    var anho: Int = 0,
    var mes: Int = 0,
    var cargoTipoId: Int? = null,
    var legajoId: Long? = null,
    var horasSemanales: BigDecimal = BigDecimal.ZERO,
    var horasTotales: BigDecimal = BigDecimal.ZERO,
    var desarraigo: Byte = 0,
    var alta: Byte = 0,
    var baja: Byte = 0,
    var cambio: Byte = 0,
    var solicitud: String? = null,
    var autorizado: Byte = 0,
    var rechazado: Byte = 0,
    var respuesta: String? = null,
    var transferido: Byte = 0,
    var curso: CursoDto? = null,
    var cargoTipo: CargoTipoDto? = null,
    var persona: PersonaDto? = null

)
